### PR TITLE
translate tutorial 0_config

### PR DESCRIPTION
**Pull request by liqikai9**

### CODE REVIEW

**Comment by jin-s13** [comment]
@liqikai9  please check the linting

File: docs_zh-CN/tutorials/0_config.md [hunk 1]
```diff
@@ -1,3 +1,235 @@
 # 教程 0: 模型配置文件
 
-内容建设中……
+我们使用python文件作为配置文件，将模块化设计和继承设计结合到配置系统中，便于进行各种实验。
```
Comment (innerlee) [inline docs_zh-CN/tutorials/0_config.md:3]: add space between cn and en characters

`我们使用 python 文件作...`

File: docs_zh-CN/tutorials/0_config.md [hunk 2]
```diff
@@ -1,3 +1,235 @@
...
+  - [配置文件命名约定](#配置文件命名约定)
+    - [配置系统](#配置系统)
+  - [常见问题](#常见问题)
+    - [在配置中使用中间变量](#在配置中使用中间变量)
+
+<!-- TOC -->
+
+## 通过脚本参数修改配置
+
+当使用 "tools/train.py" 或 "tools/test.py" 提交作业时，您可以指定 `--cfg-options` 来就地修改配置。
```
Comment (innerlee) [inline docs_zh-CN/tutorials/0_config.md:20]: ~~就地~~

File: docs_zh-CN/tutorials/0_config.md [hunk 3]
```diff
@@ -1,3 +1,235 @@
...
+- 基于热图的二维自顶向下的人体姿态估计实例
+
+    为了帮助用户对完整的配置结构和配置系统中的模块有一个基本的了解，
+    我们下面对配置文件 'https://github.com/open-mmlab/mmpose/tree/e1ec589884235bee875c89102170439a991f8450/configs/top_down/resnet/coco/res50_coco_256x192.py' 作简要的注释。
+    有关每个模块中每个参数的更详细用法和替代方法，请参阅 API 文档。
+
+    ```python
+    # 运行设置
+    log_level = 'INFO'  # 日志记录级别
+    load_from = None  # 从给定路径将模型作为预训练的模型加载。这将不会重新开始训练
```
Comment (innerlee) [inline docs_zh-CN/tutorials/0_config.md:68]: 从给定路径加载预训练模型。

Comment (innerlee) [reply docs_zh-CN/tutorials/0_config.md:68]: What does "这将不会重新开始训练" mean

File: docs_zh-CN/tutorials/0_config.md [hunk 4]
```diff
@@ -1,3 +1,235 @@
...
+        # 它们通过 `constructor` 构建，参阅 "tutorials/4_new_modules.md"
+        # 的实现。
+        type='Adam',  # 优化器的类型, 参阅 https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/optimizer/default_constructor.py#L13 获取更多细节
+        lr=5e-4,  # 学习率, 参数的详细用法见 PyTorch 文档
+    )
+    optimizer_config = dict(grad_clip=None)  # 不限制梯度的范围
+    # 学习率调整策略
+    lr_config = dict(  # 用于注册 LrUpdater 钩子的学习率调度器的配置
+        policy='step',  # 调整策略, 还支持 CosineAnnealing, Cyclic, 等等，请参阅 https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/hooks/lr_updater.py#L9 获取支持的 LrUpdater 细节
+        warmup='linear', # 使用的预热类型，它可以是None (不使用预热), 'constant', 'linear' 或者 'exp'.
```
Comment (innerlee) [inline docs_zh-CN/tutorials/0_config.md:91]: space before `None`

File: docs_zh-CN/tutorials/0_config.md (hunk 1 above)
Comment (liqikai9) [reply docs_zh-CN/tutorials/0_config.md:3]: I see. Thanks for reminding me!

File: docs_zh-CN/tutorials/0_config.md (hunk 3 above)
Comment (liqikai9) [reply docs_zh-CN/tutorials/0_config.md:68]: I think it means that the model loaded will not be trained from scratch, so I used the word "不会重新开始训练."
Actually, I was wondering how to translate the word "resume." Can I change the whole translation into "从给定路径加载预训练模型，这样，模型不会重头开始训练"? Is this appropriate?

Comment (ly015) [reply docs_zh-CN/tutorials/0_config.md:68]: 感觉不需要“这将不会重新开始训练”这句, 给人感觉比较confusing. “加载预训练模型”已经表达清楚了

Comment (liqikai9) [reply docs_zh-CN/tutorials/0_config.md:68]: 好的

**Review by innerlee — APPROVED** [review]
